py/picca/delta_extraction/data_catalogues/desi_healpix.py

```diff
@@ -186,16 +185,15 @@ def read_file(self, filename, catalogue, forests_by_targetid):
             targetid = row["TARGETID"]
             w_t = np.where(targetid_spec == targetid)[0]
             if len(w_t) == 0:
-                self.logger.warning(
-                    f"Error reading {targetid}. Ignoring object")
+                #self.logger.warning(
```

Comment:
why are you removing this comments?

Reply:
I thought I have restored it in the last commit, I had to comment it out when I was testing the parallelization, but should not be removed.